iOS-TvShows/iOS-TvShows/Application/AppDelegate.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AppDelegate.swift
+//  iOS-TvShows
+//
+//  Created by Filip Ban on 08.11.2021..
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let initialViewController = UINavigationController()
```

Comment:
Most of the comments I'll left will be regarded as you are working on a project with a team where cohesion and code style is very important :) 

Would propose better naming - `rootNavigationController` or similar which will include navigation controller in name - for devs to be aware of correct type.

Reply:
Of course, left it cos this is the first thing I did and then figured I need UINav for wireframe, should be more careful next time